### ISSUE TITLE
[$30][MSFT-235] [Challenge Feed][NVD][Edge] About • Contact • Help • Privacy • Terms read-out differently in Edge by the screen-readers

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2892]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges
2. Hover over 'About', check the voice of the screen-readers
3. Hover over 'Contact', check the voice of the screen-readers
4. Hover over 'Help', check the voice of the screen-readers
5. Hover over 'Privacy', check the voice of the screen-readers
6. Hover over 'Terms', check the voice of the screen-readers

**Expected Result**
Should read-out by the screen-readers without an issue

**Actual Result**
About • Contact • Help • Privacy • Terms read-out differently in Edge by the screen-readers

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Edge

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F71

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096038 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096038 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096038 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@codeMinter PR https://github.com/topcoder-platform/community-app/pull/2969

**Comment by anonymousjaggu** [comment]
@PrakashDurlabhji please check the comment in PR.

**Comment by PrakashDurlabhji** [comment]
@anonymousjaggu PR updated

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096755 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096755 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@anonymousjaggu PR updated #2969

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096755 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@anonymousjaggu taking it back to me as it was already with me before and here is video of verification

**Comment by PrakashDurlabhji** [comment]
@anonymousjaggu updated PR https://github.com/topcoder-platform/community-app/pull/2969 and video is in PR.

**Comment by tosha5252** [comment]
Works in Edge but not in Chrome...

OS: Windows 10
Browsers: Edge and Chrome

**Comment by PrakashDurlabhji** [comment]
@tosha5252 I guess it never worked in chrome before, will see the issue of chrome, But requirement of this issue is already met I feel.

**Comment by tosha5252** [comment]
@PrakashDurlabhji Chrome coverage is important...so we can't pass it if it is tested in Chrome browser and not working

**Comment by tosha5252** [comment]
[TinyTake by MangoApps-29-07-2019-07-53-34.zip](https://github.com/topcoder-platform/community-app/files/3442307/TinyTake.by.MangoApps-29-07-2019-07-53-34.zip)

This is what I am seeing in Chrome

**Comment by lakshmiathreya** [comment]
This is working as expected in Chrome - latest version. Could you double check @tosha5252. fyi @nkumar @sushil

**Comment by PrakashDurlabhji** [comment]
@tosha5252 Request you to resolve discussion at earliest because people are waiting to just take my issues and copy PR rather than picking up new tickets.

watching QA fail label people have started directly unassigning me hence afraid if even this is taken suddenly.

@sushilshinde

**Comment by sushilshinde** [comment]
@PrakashDurlabhji don't panic, nobody will take your issues. Please focus on open issues, there are many.
Issues that were not active have re-assigned.

**Comment by lakshmiathreya** [comment]
Moving to Pass - verified in Chrome and Edge ... fyi @tosha5252 @drasticdpk

**Comment by lakshmiathreya** [comment]
Verified :
-- Edge - MacOS/Voice Over, Win10/NVDA
-- Chrome - MacOS/Voice Over, Win10/NVDA

Closing the Issue.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096755<br/><br/>```This is an automated message for crazyk via Topcoder X```
